### PR TITLE
[WIP] Add fips support for amq-streams scenario

### PR DESCRIPTION
**Pull request by vi-patel**

### CODE REVIEW

**Comment by vi-patel** [comment]
/pj-rehearse amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.17-lp-interop

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.17-lp-interop-amq-streams-2-7-fips

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.17-lp-interop-amq-streams-2-7-fips

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.17-lp-interop-amq-streams-2-7-fips

**Comment by dfrazzette** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.17-lp-interop-amq-streams-2-7-fips

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.17-lp-interop-amq-streams-2-7-fips

**Comment by vi-patel** [comment]
/pj-rehearse periodic-ci-amq-streams-qe-strimzi-kafka-operator-main-amq-streams-ocp4.17-lp-interop-amq-streams-2-7-fips

File: ci-operator/config/amq-streams-qe/strimzi-kafka-operator/.config.prowgen [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  job_states_to_report:
+  - success
+  - failure
+  - error
+  report_template: '{{if eq .Status.State "success"}} :slack-green: Job *{{.Spec.Job}}*
+                    ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :failed:
+                    Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
+                    logs> {{end}}'
+  job_names:
+  - amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.17-lp-interop
```
Comment (dfrazzette) [inline ci-operator/config/amq-streams-qe/strimzi-kafka-operator/.config.prowgen:12]:
```suggestion
  - amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.17-lp-interop
  - amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.17-lp-interop-fips
```

**Comment by vi-patel** [comment]
Closing in favor of updated PR: https://github.com/openshift/release/pull/55669

**Pull request closed without merging**